src/apps/Permissions/AppPermissions.js

```diff
@@ -107,9 +107,11 @@ class Row extends React.Component {
     if (entity.type === 'app') {
       return <AppInstanceLabel app={entity.app} proxyAddress={entity.address} />
     }
+
     return (
-      <IdentityBadge
+      <LocalIdentityBadge
         entity={entity.type === 'any' ? 'Any account' : entity.address}
+        address={entity.address}
```

Comment:
Seems like in aragon/aragon as well, we could just use `entity` (and do an `isAddress` check on it before resolving) in `LocalIdentityBadge`.